README.md

```diff
@@ -83,7 +83,6 @@ User can add a STM32 based board following this [wiki](https://github.com/stm32d
 
 
 ### [Nucleo 144](https://www.st.com/content/st_com/en/products/evaluation-tools/product-evaluation-tools/mcu-eval-tools/stm32-mcu-eval-tools/stm32-nucleo-boards.html) boards
-
```

Comment:
still one line deleted which should not.

Reply:
sorry about that! will fix now